frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -100,37 +149,142 @@ describe("[OSS] embedding settings", () => {
       });
     });
   });
+
+  it("should link to quickstart for interactive embedding", async () => {
+    await setupEmbedding({
+      settingValues: {
+        "enable-embedding": false,
+        version: { tag: "v0.49.3" },
+      },
+    });
+    expect(getQuickStartLink()).toBeInTheDocument();
+    expect(getQuickStartLink()).toHaveProperty(
+      "href",
+      "https://www.metabase.com/docs/v0.49/embedding/interactive-embedding-quick-start-guide.html?utm_source=oss&utm_media=embed-settings",
+    );
+  });
+
+  it("should redirect users back to embedding settings page when visiting the full-app embedding page when embedding is not enabled", async () => {
+    await setup({
+      settings: [createMockSettingDefinition({ key: "enable-embedding" })],
+      settingValues: createMockSettings({ "enable-embedding": false }),
+      initialRoute: FULL_APP_EMBEDDING_URL,
+    });
+
+    expect(screen.getByText("Interactive embedding")).toBeInTheDocument();
+    expect(screen.getByText(/Embed dashboards, questions/)).toBeInTheDocument();
+  });
+
   describe("when the embedding is enabled", () => {
-    it("should allow going to static embedding settings page", async () => {
-      const { history } = await setupEmbedding({
-        settingValues: { "enable-embedding": true },
+    let history: Awaited<ReturnType<typeof setupEmbedding>>["history"];
+
+    beforeEach(async () => {
+      history = (
+        await setupEmbedding({
+          settingValues: { "enable-embedding": true },
+        })
+      ).history;
+    });
+
+    describe("static embedding", () => {
+      it("should allow going to static embedding settings page", async () => {
+        await goToStaticEmbeddingSettings();
+
+        const location = history.getCurrentLocation();
+        expect(location.pathname).toEqual(staticEmbeddingSettingsUrl);
       });
 
-      await goToStaticEmbeddingSettings();
+      it("should show info about static embedding", async () => {
+        const withinStaticEmbeddingCard = within(
+          screen.getByRole("article", {
+            name: "Static embedding",
+          }),
+        );
 
-      const location = history.getCurrentLocation();
-      expect(location.pathname).toEqual(staticEmbeddingSettingsUrl);
+        expect(
+          withinStaticEmbeddingCard.getByRole("heading", {
+            name: "Static embedding",
+          }),
+        ).toBeInTheDocument();
+        expect(
+          withinStaticEmbeddingCard.getByText(/Use static embedding when/),
+        ).toBeInTheDocument();
+      });
+
+      it("should not allow access to the static embedding settings page", async () => {
```

Comment:
Should this be "should allow access" instead? As in this test we can see the "Embedding secret key"

```suggestion
      it("should allow access to the static embedding settings page", async () => {
```

Reply:
Fixed in 7ebbf42f03b871d41a47503657f546a2215ffea4